### DISCUSSION TITLE
BRouter profile selection

### DISCUSSION THREAD

**Discussion by Leeb64** [devemux86/cruiser#230] [Q&A] [answered]
Getting to the BRouter profile selection is a bit cumbersome at the moment....
Settings/Routes/Options/BRouter(offline)/Advanced/BRouter profile...
Wouldn't it be possible to allow some shortcut?
Maybe under Options?
Profile: INT<->EXT

**Comment by devemux86** [comment, marked as answer]
Settings have search (at top in first screen) for this purpose: go immediately to the desired setting.

> Wouldn't it be possible to allow some shortcut?

Thanks for the suggestion. Where exactly do you mean?

Non power users do not know about external profiles,
so better not confuse them with options unless they are advanced?

<img src="https://github.com/devemux86/cruiser/assets/3484020/32f4f726-a274-4dcb-a7fc-aff95368c660" width="50%"/><img src="https://github.com/devemux86/cruiser/assets/3484020/903a28ff-5ca3-40c7-b230-47dd83f292d8" width="50%"/>
